Resources/Private/JavaScript/asset-upload/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "main": "src/index.ts",
   "dependencies": {
-    "@media-ui/core": "workspace:*"
+    "@media-ui/core": "workspace:*",
+    "@media-ui/media-module": "workspace:*"
```

Comment:
I try to prevent circular dependencies, therefore the `Property` component should be moved into `@media/core` to get rid of the dependency on the `media-module`.

Reply:
I move the property component to `@media/core`